### PR TITLE
Removes all ZEN_MOD_* statements

### PR DESCRIPTION
**Pull request by tarrenj**
Done with: `find . -name .git -prune -o -type f -exec sed -i -r -e '/ZEN_MOD(_START|_STOP|_END|-END)/d' {} +`

### CODE REVIEW

**Comment by tarrenj** [comment]
Requested by @albertog78.
Should not be merged until features currently under review are merged (namely `delaypenalty`), and `zen_mod_Removal` is resynced with `development`

**Comment by tarrenj** [comment]
Merged `development` into `zen_mod_Removal`.

File: src/rpcrawtransaction.cpp [hunk 1]
```diff
@@ -166,7 +166,6 @@ void TxToJSON(const CTransaction& tx, const uint256 hashBlock, UniValue& entry)
 }
 
 UniValue getrawtransaction(const UniValue& params, bool fHelp)
-{   // ZEN_MOD_START
```
Comment (cronicc) [inline src/rpcrawtransaction.cpp:169]: don't remove bracket

**Review by cronicc — CHANGES_REQUESTED** [review]

File: src/rpcrawtransaction.cpp (hunk 1 above)
Comment (tarrenj) [reply src/rpcrawtransaction.cpp:169]: ACK, testing for more issues before pushing.

**Review by cronicc — APPROVED** [review]

**Review by albertog78 — APPROVED** [review]

**Pull request merged**